data-raw/healthy-lives/behavioural-risk-factors/drug-misuse.R

```diff
@@ -0,0 +1,52 @@
+# ---- Load packages ----
+library(tidyverse)
+library(readxl)
+library(httr)
+
+# ---- Scrape URL for drug deaths ----
+# ---- Source: https://www.ons.gov.uk/peoplepopulationandcommunity/birthsdeathsandmarriages/deaths/datasets/drugmisusedeathsbylocalauthority ----
+drug_url <- "https://www.ons.gov.uk/file?uri=/peoplepopulationandcommunity/birthsdeathsandmarriages/deaths/datasets/drugmisusedeathsbylocalauthority/current/2022localauthorities.xlsx"
+
+# ---- Download and read URL as temp file ----
+drug_temp_file <- tempfile(fileext = ".xlsx")
+download.file(drug_url, drug_temp_file, mode = "wb")
+drug_data <- read_excel(drug_temp_file)
+
+# ---- Scrape URL for population ----
+# ---- Source: https://www.ons.gov.uk/peoplepopulationandcommunity/populationandmigration/populationestimates/datasets/estimatesofthepopulationforenglandandwales
+population_url <- "https://www.ons.gov.uk/file?uri=/peoplepopulationandcommunity/populationandmigration/populationestimates/datasets/estimatesofthepopulationforenglandandwales/mid20222023localauthorityboundaires/mye22tablesew2023geogsv2.xlsx"
+
+# ---- Download and read URL as temp file ----
+population_temp_file <- tempfile(fileext = ".xlsx")
+download.file(population_url, population_temp_file, mode = "wb")
+population_data <- read_excel(population_temp_file)
+
+# ---- Reduce datasets to only include Wales ----
+drug_wales <- read_excel(drug_temp_file, sheet = "Table 1", range = "A4:E432") |>
```

Comment:
Could you please add a couple of lines of comment defining/explaining what the data is